webserver/pkgpkr/requirements.txt

```diff
@@ -0,0 +1,6 @@
+psycopg2
```

Comment:
Probably should freeze these versions so we don't end up with issues if one of our dependencies does a major upgrade, etc.

```
boto3==1.9.224
...
```